### DISCUSSION TITLE
歌曲乱跳

### DISCUSSION THREAD

**Discussion by ndbcd** [HyPlayer/HyPlayer#146] [Q&A]
启动播放时歌曲会乱套，一会播这首，一会播那首。

**Comment by kengwang** [comment]
可否提供当前播放的歌单链接以便调试
同时也麻烦提供下 当前播放模式, 是否开启不重复循环, 是否是 VIP

**Comment by ApigeonAPE** [comment]
这种多半是自己开了不重复循环，而且播放的又是vip歌曲



在 2022-03-12 19:11:54，"Kengwang" ***@***.***> 写道：

可否提供当前播放的歌单链接以便调试
同时也麻烦提供下 当前播放模式, 是否开启不重复循环, 是否是 VIP

—
Reply to this email directly, view it on GitHub, or unsubscribe.
Triage notifications on the go with GitHub Mobile for iOS or Android.
